--- Main.py	2022-10-27 17:40:08.000000000 +0200
+++ Main.py.new	2022-11-17 17:39:19.235880380 +0100
@@ -403,7 +403,7 @@
                                 bufsize=0, close_fds=True, shell=False)
                 logger = subproc.stdin
             elif isinstance(self.logfile, str):
-                logger = open(self.logfile, "a+", 0)
+                logger = open(self.logfile, "a+b", 0)
             else:
                 raise TypeError("'logfile' must be a string or array")
             os.dup2(logger.fileno(), sys.stdout.fileno())
